### PR TITLE
feat: add stats json output

### PR DESCRIPTION
**Pull request by wei**
- Add a `stats` output that mimics the response of `/probot/stats` endpoint. https://github.com/probot/probot/blob/1c31415759b2f0e03e175ddb3fd51a402f2b77c7/src/apps/stats.ts#L14

- Remove `suspendedInstallations` output which does not exist in the codebase.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,8 +23,8 @@ jobs:
           id: ${{ secrets.APP_ID }}
           private_key: ${{ secrets.PRIVATE_KEY }}
       - run: "echo installations: '${{ steps.stats.outputs.installations }}'"
-      - run: "echo suspended: '${{ steps.stats.outputs.suspendedInstallations }}'"
```
Comment (gr2m) [inline README.md:26]: Why did you remove this?

https://github.com/gr2m/app-stats-action/blob/1f49f3ba2e3ac8d66f7b5e26182d9d44ee3f053b/lib/app-stats.js#L33-L38

File: README.md [hunk 2]
```diff
@@ -23,8 +23,8 @@ jobs:
...
       - run: "echo most popular repositories: '${{ steps.stats.outputs.popular_repositories }}'"
+      - run: "echo stats: '${{ steps.stats.outputs.stats }}'"
```
Comment (gr2m) [inline README.md:27]: I don't think we need to add a `stats` output that just combines `installations` and `popular_repositories`. You can do that yourself as consumer of the action.

This is too Probot-specific. I created this action as a path to get stats for Probot users, but it's useful to everyone building GitHub Apps

**Comment by wei** [comment]
@gr2m Thanks for the review, the output `suspendedInstallations` was not set here, maybe just need to add a line.

https://github.com/gr2m/app-stats-action/blob/d11333ef309852dff4a49bd1e676de61ea95eb86/index.js#L18-L19

Closing for now.

**Comment by gr2m** [comment]
> the output suspendedInstallations was not set here, maybe just need to add a line

oh indeed, yes! will fix!
